app/validators/metadata_presenter/date_validator.rb

```diff
@@ -1,13 +1,18 @@
 module MetadataPresenter
   class DateValidator < BaseValidator
     DATE_STRING_VALIDATIONS = %w[date_after date_before].freeze
+    YEAR_LOWER_BOUND = 1000
+    YEAR_UPPER_BOUND = 2099
```

Comment:
can we do 3000? Just in case someone has a form for dealing with long land leases or something :D

Reply:
Oh yeah that makes sense!
Will change in a moment. I need to raise another PR in the filestore to go along with these changes 👍